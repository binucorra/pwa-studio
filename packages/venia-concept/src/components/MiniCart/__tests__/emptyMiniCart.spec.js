import React from 'react';
import { createTestInstance } from '@magento/peregrine';

import EmptyMiniCart from '../emptyMiniCart';

jest.mock('../trigger');

test('renders the product list when appropriate', () => {
    const tree = createTestInstance(
        <EmptyMiniCart />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});