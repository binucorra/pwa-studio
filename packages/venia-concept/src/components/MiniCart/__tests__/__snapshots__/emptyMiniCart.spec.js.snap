// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the product list when appropriate 1`] = `
<div>
  <h3>
    There are no items in your shopping cart
  </h3>
  ( Trigger Component Here )
</div>
`;
